_config.yml

```diff
@@ -288,3 +288,6 @@ defaults:
       comments: # true
       share: false
       related: false
+
+theme: jekyll-theme-midnight
```

Comment:
Does this theme work for you? It doesn't change anything on my end. If it doesn't do anything, can you remove this line?